doc/developer-guides/hld/hostbridge-virt-hld.rst

```diff
@@ -20,11 +20,10 @@ There is PCI host bridge emulation in DM. The bus hierarchy is determined by ``a
         -s 5,virtio-console,@stdio:stdio_port \
         -s 6,virtio-hyper_dmabuf \
         -s 3,virtio-blk,/home/acrn/UserVM.img \
-        -s 4,virtio-net,tap0 \
+        -s 4,virtio-net,tap=0,mac_seed=$mac_seed \
```

Comment:
That means the tap device would be called `0`, which is a rather odd name for it, let's keep it at `tap0`, i.e.:
```suggestion
        -s 4,virtio-net,tap=tap0,mac_seed=$mac_seed \
```

Reply:
Sorry, Failed to change to "tap=tap0".
The virtio-net module will use snprintf to merge the type and name to the full device name.
eg.  type is tap, name is LaaG, the full device name is tap_LaaG.
If the tap0 is a confused name, I think it's better to use the tap=LaaG to show how to set virtio-net.

Reply:
Does that mean that the user has to create a TAP device called `tap_XXX` and then pass `tap=XXX` as an argument to `acrn-dm`??

Reply:
We have not change the create device logic.
So if device has been created and name is tap_XXX, the type is "tap".